submissions/Ant-C-tech/memory-pair-game/js/app.js

```diff
@@ -0,0 +1,278 @@
+
+        const cardBack = document.createElement('img')
+        cardBack.setAttribute('src', 'img/card-back2.jpg')
+        cardBack.setAttribute('alt', 'japanese ornament')
+        cardBack.classList.add('cardBack')
+
+        const cardFace = document.createElement('div')
+        cardFace.classList.add('cardFace')
+        const japText = document.createElement('p')
+        japText.innerText = cardContent[0].jap
+        japText.classList.add('japText')
+        const engText = document.createElement('p')
+        engText.innerText = cardContent[0].eng
+        engText.classList.add('engText')
+        cardFace.append(japText, engText)
+        card.append(cardBack, cardFace)
+
+        card.addEventListener('click', () => {
+            card.classList.add('main-notActive')
+            card.classList.add('card-rotate')
+            cardBack.classList.add('cardBack-rotate')
+            cardBack.addEventListener('transitionend', function() {
+                cardFace.classList.add('cardFace-rotate')
+            }, { once: true })
+            checkAnswer()
+        })
+        gameField.appendChild(card)
+    }
+    return gameField
+}
+
+function createCongratulationScr() {
+    const congratulationWrapper = new DocumentFragment()
+
+    const congratulationIco = document.createElement('img')
+    congratulationIco.setAttribute('src', 'img/lucky-cat-toy-svgrepo-com.svg')
+    congratulationIco.setAttribute('alt', 'mentor')
+    congratulationIco.classList.add('congratulationIco')
+
+    const congratulation = document.createElement('h3')
+    congratulation.textContent = 'Congratulation!!!'
+    congratulation.classList.add('congratulation')
+
+    const congratulationText = document.createElement('p')
+    congratulationText.innerHTML = `It took you <span class="attempts">${attempts}</span> attempts!`
+    congratulationText.classList.add('congratulationText')
+
+    const congratulationSubText = document.createElement('p')
+    congratulationSubText.textContent = `Now, you maybe feel yourself true ninja or maybe even samurai... But remember...`
+    congratulationSubText.classList.add('congratulationSubText')
+
+    const newGameBtn = document.createElement('h3')
+    newGameBtn.textContent = 'A samurai has no goal, only a path...'
+    newGameBtn.classList.add('newGame')
+
+    congratulationWrapper.append(congratulationIco, congratulation, congratulationText, congratulationSubText, newGameBtn)
+
+    newGameBtn.addEventListener('click', function() {
+        TASK = []
+        attempts = 0
+        setTask()
+        window.addEventListener('resize', phoneAdaptation)
+        phoneAdaptation()
+        changeContent(createGameField())
+    }, { once: true })
+
+    return congratulationWrapper
+}
+
+function changeContent(content, callback) {
+    MAIN.classList.add('main-hide')
+    MAIN.addEventListener('transitionend', function() {
+        MAIN.innerHTML = ''
+        MAIN.append(content)
+        if (callback) {
```

Comment:
You can use this shortcut as well
```suggestion
        callback && callback()
```